doc/man1/openssl-req.pod.in

```diff
@@ -33,9 +33,12 @@ B<openssl> B<req>
 [B<-config> I<filename>]
 [B<-section> I<name>]
 [B<-x509>]
+[B<-CA> I<filename>|I<uri>]
```

Comment:
I can only repeat that I am really not too fond of this added functionality as it is just a duplicate of the same functionality in x509, duplicating practically the same code.

Reply:
I needed those options in the tests, and the alternative would have been to use `x509`, which on the other hand lacks `-addext`.
There was already strong overlap between the two apps, as mentioned in #13720.

Reply:
OK, you convinced me. :grin: